docs/pages/enroll-resources/mcp-access/stdio.mdx

```diff
@@ -112,7 +112,8 @@ EOF
 
 (!docs/pages/includes/create-role-using-web.mdx!)
 
-Create the Teleport user assigned the `access-mcp` role we've just created:
+Create a Teleport user assigned to the `access-mcp` role we've just created,
+assigning <Var name="my_user" /> to the name of your new Teleport user:
 
 ```code
 $ tctl users add --roles=access-mcp my_user
```

Comment:
```suggestion
$ tctl users add --roles=access-mcp <Var name="my_user" />
```

Reply:
This line was also changed in a conflicting commit